operator/pkg/controllers/etcd/pod.go

```diff
@@ -115,6 +116,19 @@ func podSpecFor(controlPlane *v1alpha1.ControlPlane) *v1.PodSpec {
 					},
 				},
 			}},
+			LivenessProbe: &v1.Probe{
+				Handler: v1.Handler{
+					HTTPGet: &v1.HTTPGetAction{
```

Comment:
Where do these values come from?

Reply:
Not sure how useful this is: https://github.com/coreos/etcd-operator/blob/13a1831f9744207b39733bbe91413ca2e389aa40/pkg/util/k8sutil/pod_util.go#L71

Reply:
We're also using HTTP probes where we need just need to specify a host and endpoint and need a successful HTTP status code. With command probes, we could use curl or etcdctl but those probes rely on a non-zero exit code so with curl we could have an exit code of zero but get returned a 400 and still need to check the content of what's returned from the endpoint.